src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
This nested loop is used to check for duplicate values in the object. However, this is not efficient because it results in a time complexity of O(n^2). A more efficient way would be to check for duplicates while inverting the object in the second loop. If a duplicate is found, you can return null immediately.